src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/ClaimRepository.java

```diff
@@ -68,6 +68,12 @@
     @SqlQuery(SELECT_FROM_STATEMENT + " WHERE claim.is_migrated = false")
     List<Claim> getAllNotMigratedClaims();
 
+    @SqlQuery(
+        SELECT_FROM_STATEMENT + " WHERE claim.response ->>'freeMediation = 'yes' " +
+            "AND claim.claimant_response ->>'freeMediation = 'yes'" +
+            "AND claim.claimant_responded_at = :claimant_responded_date")
+    List<Claim> getMediationClaimsForDate(@Bind("claimant_responded_date") LocalDate claimant_responded_date);
```

Comment:
Can give the java variable a `camelCase` name; the `@Bind` value is what needs to match the `@SqlQuery` placeholder.

Reply:
done